sinch/core/adapters/requests_http_transport.py

```diff
@@ -8,7 +8,7 @@
 class HTTPTransportRequests(HTTPTransport):
     def __init__(self, sinch):
         super().__init__(sinch)
-        self.session = requests.Session()
+        self.http_session = requests.Session()
```

Comment:
Naive question: no need to do same things for this class. Not async, so we are sure there is no multiple session created ?

Reply:
I've changed this code because I've wanted to align properties naming across HTTP implementations. `Requests` docs also suggest using HTTP connection pooling.